simple_salesforce/api.py

```diff
@@ -299,16 +302,16 @@ def restful(self, path, params=None, method='GET', **kwargs):
         * method: HTTP request method, default GET
         * other arguments supported by requests.request (e.g. json, timeout)
         """
-
-        url = self.base_url + path
+        base_regex = r'\/*services\/data\/v[0-9]{2}.[0-9]{1,}\/'
```

Comment:
this is something that you really should consider to merge since resources URIs include "services/data/vXX.X" in it, so if any tries to search any resource using the URI returned in a previous call it will not work as it now assumes resources URIs dont include the services/data/vXX.X part.

Reply:
Not opposed to merging this in just need to review it.